### PR TITLE
catches error 99 and returns original buffer

### PR DESCRIPTION
**Pull request by davidfig**
This is a pull request to fix issue #32. It catches the error-99 and returns the original buffer.

### CODE REVIEW

**Comment by shinnn** [comment]
https://travis-ci.org/imagemin/imagemin-pngquant/jobs/192241879#L833-L837

**Comment by davidfig** [comment]
I fixed formatting and added a test case for error 99.

File: test.js [hunk 1]
```diff
@@ -37,3 +37,10 @@ test('throw on corrupt image', async t => {
 	const buf = await fsP.readFile(path.join(__dirname, 'fixture-corrupt.png'));
 	t.throws(m()(buf), /PNG file corrupted/);
 });
+
+test('return original buffer on err.code===99 (not compressible at quality requested)', async t => {
+	const buf = await fsP.readFile(path.join(__dirname, 'fixture-no-compress.png'));
+	const data = await m({quality: 100})(buf);
+	t.true(data.length === buf.length);
```
Comment (shinnn) [inline test.js]:
```js
t.is(data.length, buf.length);
```

File: test.js [hunk 2]
```diff
@@ -37,3 +37,10 @@ test('throw on corrupt image', async t => {
 	const buf = await fsP.readFile(path.join(__dirname, 'fixture-corrupt.png'));
 	t.throws(m()(buf), /PNG file corrupted/);
 });
+
+test('return original buffer on err.code===99 (not compressible at quality requested)', async t => {
```
Comment (shinnn) [inline test.js]: Somewhat redundant... for example `'skip optimizing a fully optimized PNG'` ?

**Comment by shinnn** [comment]
Great work, @davidfig! I added some more comments.

**Comment by zeh** [comment]
Thanks for this. Was just fixing the same thing and just realized it already had a PR and a commit for it!

**Comment by Zephir77167** [comment]
Hi, as @zeh said, a version bump would be greatly appreciated. Same issue, image-webpack-loader depends on it.

**Pull request merged**